src/common.cpp

```diff
@@ -25,6 +25,11 @@
 #include <psapi.h>
 #endif
 
+#if defined(GB_SYSTEM_OPENBSD)
+#include <sys/resource.h>
+#include <sys/wait.h>
```

Comment:
only `sys/wait.h` should be required for `WIFEXITED` and `WEXITSTATUS` macro.

also, I would put the missing includes inside `src/gb/gb.h` under the `#if defined(GB_SYSTEM_OPENBSD)`